blog/2023/12-11-roku-v200/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Roku Version 2.0.0
+description: I Want a Rokupotamus for Christmas
+authors: 1hitsong
+date: 2023-12-11
```

Comment:
You don't need to set the date in the file, Docusaurus just infers it from the folder/file name.